### PR TITLE
Update to google-gax 0.17.1

### PR DESCRIPTION
**Pull request by kolodny**
- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const common = require('@google-cloud/common');
 const extend = require('extend');
 const GrpcService = require('@google-cloud/common-grpc').Service;
 const googleAuth = require('google-auto-auth');
-const grpc = require('google-gax').grpc().grpc;
+const gax = require('google-gax');
```
Comment (JustinBeckwith) [inline src/index.js]: Not sure if you use gax elsewhere in the file, but you could shorten this all to `const {GrpcClient} = require('google-gax');`

**Comment by sduskis** [comment]
TypeError: gax.GrpcClient is not a constructor

File: src/index.js (hunk 1 above)
Comment (kolodny) [reply src/index.js]: Good call, changed

**Comment by sduskis** [comment]
@kolodny, did you manually change generated code?

**Comment by kolodny** [comment]
@sduskis I did, @JustinBeckwith will that be a problem when the code is regenerated?

**Comment by stephenplusplus** [comment]
@JustinBeckwith @crwilcox how would we get this change into the generator? I'm blocked on this for https://github.com/googleapis/nodejs-spanner/pull/220 as well.

**Comment by JustinBeckwith** [comment]
@alexander-fenster has the latest on this :)

**Pull request closed without merging**